sdk/resourcemanager/Azure.ResourceManager.Core/tests/Scenario/ResourceGroupsRestOperationsTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Core.Tests
+{
+    public class ResourceGroupsRestOperationsTests : ResourceManagerTestBase
```

Comment:
we shouldn't need rest operation tests directly.  Coverage should be achieved via the Operations, Container or ParentOperations